app-modules/contact/tests/Organization/ViewOrganizationTest.php

```diff
@@ -0,0 +1,88 @@
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use App\Models\User;
+
+use function Pest\Laravel\actingAs;
+
+use AidingApp\Contact\Models\Contact;
+use AidingApp\Contact\Models\Organization;
+use AidingApp\Contact\Filament\Resources\OrganizationResource;
+
+test('View Organization is gated with proper access control', function () {
+    $user = User::factory()->licensed(Contact::getLicenseType())->create();
+
+    $organization = Organization::factory()->create();
+
+    actingAs($user)
+        ->get(
+            OrganizationResource::getUrl('view', [
+                'record' => $organization,
+            ])
+        )->assertForbidden();
+
+    $user->givePermissionTo('organization.view-any');
+    $user->givePermissionTo('organization.*.view');
+
+    actingAs($user)
+        ->get(
+            OrganizationResource::getUrl('view', [
+                'record' => $organization,
+            ])
+        )->assertSuccessful();
+});
+test('View Organization table action gated with proper access control', function () {
+    $user = User::factory()->licensed(Contact::getLicenseType())->create();
+
+    $organization = Organization::factory()->create();
+
+    actingAs($user)
+        ->get(
+            OrganizationResource::getUrl('view', [
+                'record' => $organization,
+            ])
+        )->assertForbidden();
+
+    $user->givePermissionTo('organization.view-any');
+    $user->givePermissionTo('organization.*.view');
+
+    actingAs($user)
+        ->get(
+            OrganizationResource::getUrl('view', [
+                'record' => $organization,
+            ])
+        )->assertSuccessful();
+});
```

Comment:
This looks like it is the same test as above. Should it be removed?

Reply:
done